### ISSUE TITLE
Queue Constants

### ISSUE THREAD

**Issue by OwenVey** [taycaldwell/riot-api-java#107]
Is there a plan to add constants for Queue ID's like you have for Platform and Region constants?

**Comment by taycaldwell** [comment]
Constants for queues existed before. Had to do some digging to see why they were removed.

Looks like they were removed with the update to support League v3, as the methods now take a String representation of the queue type rather than the enum.

https://github.com/taycaldwell/riot-api-java/commit/2aa282822136f0ebdaa3905913cbca4a27282438

I know some devs found these enums to be helpful for comparison of returned String values from the API, so I'm not sure I necessarily agree with this recent change.

I guess I will ask you - would you find the addition of these constants to be useful, and if so, why?

**Comment by OwenVey** [comment]
I think that these methods would only benefit devs as it allows them to access the constants quickly without having to refer to the Game Constants Documentation to find the ID of a certain Queue.

**Comment by taycaldwell** [comment]
Right. It would be ideal for devs using the library to not have to refer to external documentation or the Dev Portal to get this information.

**Comment by Linnun** [comment]
When converting to the V3 API, I basically kicked out the V1/V2 methods and the then unused enums. There is no particular reason why we should not (re)invent a few enums wherever it's appropriate.

Since the queue type from leagues does **NOT** neccessarily match actual queue types, this definitely should be an endpoint specific enum, offering only values that are actually used for league queue types.

**Comment by Linnun** [comment]
I have added a LeagueQueue constant specific to the league endpoint: https://github.com/taycaldwell/riot-api-java/commit/79189dafec2dfb07787d2c2439280a9af2a7d0d5
